types/foundry-vtt-api/types/pixi/helpers/controlicon.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * A generic helper for drawing a standard Control Icon
+ */
+declare class ControlIcon extends PIXI.Container {
+	// @TODO: Declare
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/pixi/helpers/controlicon.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.